arch/riscv/src/csr/mod.rs

```diff
@@ -259,6 +282,20 @@ pub const CSR: &CSR = &CSR {
     pmpaddr62: ReadWriteRiscvCsr::new(),
     pmpaddr63: ReadWriteRiscvCsr::new(),
 
+    hpmcounter3: ReadWriteRiscvCsr::new(),
+    hpmcounter4: ReadWriteRiscvCsr::new(),
+    hpmcounter5: ReadWriteRiscvCsr::new(),
+    hpmcounter6: ReadWriteRiscvCsr::new(),
+    hpmcounter7: ReadWriteRiscvCsr::new(),
+    hpmcounter8: ReadWriteRiscvCsr::new(),
+    hpmcounter9: ReadWriteRiscvCsr::new(),
+    hpmcounter10: ReadWriteRiscvCsr::new(),
+    hpmcounter11: ReadWriteRiscvCsr::new(),
+    hpmcounter12: ReadWriteRiscvCsr::new(),
+    hpmcounter13: ReadWriteRiscvCsr::new(),
+    hpmcounter14: ReadWriteRiscvCsr::new(),
+    hpmcounter15: ReadWriteRiscvCsr::new(),
```

Comment:
What about `hpmcounter*h`?

Reply:
I was only using these on 64-bit. I can add 32-bit too if you would find it useful? I just saw no reason to remove them as I bundled commits, they are not strictly need here.

Reply:
If they aren't used they should probably just be removed. Then 32/64 bit versions can be added in the future

Reply:
Removed